File: evals/expo-modules/04-rn-expo-modules-kotlin-name-match/reference/modules/WeatherInlineModule.kt
package expo.modules.weatherinline

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class WeatherInlineModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("WeatherInlineModule")
    Function("unit") {
      "celsius"
    }
  }
}

File: evals/expo-modules/03-rn-expo-modules-swift-name-match/reference/modules/WeatherInlineModule.swift
import ExpoModulesCore

public class WeatherInlineModule: Module {
  public func definition() -> ModuleDefinition {
    Name("WeatherInlineModule")

    Function("unit") {
      return "celsius"
    }

    Function("currentTemperature") {
      return 21
    }
  }
}
